.circleci/config.yml

```
@@ -13,6 +13,6 @@ jobs:
       - run:
           name: Build
           command: |
-            python setup.py sdist --formats=zip,gztar
+            python setup.py bdist_wheel sdist --formats=zip,gztar
```

```suggestion
            python setup.py bdist_wheel --universal sdist --formats=zip,gztar
```
keeping py2 support (with the wheel) until we explicitly drop it from this package.

---

Done in https://github.com/plotly/dash-enterprise-auth/pull/18/commits/521391c6106b19ec1445a4fc855e1dd69d7ea7e5